### PR TITLE
Changes for new Raidiam account

### PR DESCRIPTION
**Pull request by kipparker**
These changes go with the PR for updating the harvester

- Change to a simpler folder layout
- Rename library to icebreakerone.trust
- Update the certificate chain to match the new account

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -1,28 +1,28 @@
-Python API : ib1.openenergy.support
+Python API : icebreakerone.trust
```
Comment (kipparker) [inline docs/api.rst:1]: Renamed from ib1.openenergy.support to icebreakerone.trust

File: docs/ec2/directory_1.png
Comment (kipparker) [inline docs/ec2/directory_1.png:1]: Move docs from /src/docs to /docs

File: icebreakerone/trust/__init__.py
Comment (kipparker) [inline icebreakerone/trust/__init__.py:1]: Library src files moved from /src/python/openergy to icebreakerone/

**Review by croesus — APPROVED** [review]
Looks good to me

**Pull request merged**